webapp/components/test-search.html

```diff
@@ -86,50 +86,36 @@
         this.onBlur = this.handleBlur.bind(this);
       }
 
-      computeAutocompleteURL(testRuns, q) {
-        if (!testRuns || !testRuns.length) {
-          return '';
-        }
-        let url = new URL('/api/autocomplete', window.location);
-        url.searchParams.set('limit', 10);
-        url.searchParams.set(
-          'run_ids',
-          testRuns.map(r => r.id.toString()).join(','));
-        if (q) {
-          url.searchParams.set('q', q);
-        }
-        return url;
-      }
-
       ready() {
         super.ready();
-        this.q = this.query;
+        this._createMethodObserver('pathsUpdated(query, testPaths)');
```

Comment:
This sounds like a Polymer implementation detail. Is this the recommended way of creating an observer with no explicit output property?

Reply:
Yes, it's essentially an observer of multiple properties (with no output property).